mantis-server/mantis-server-worker/src/test/java/io/mantisrx/server/worker/HeartbeatTest.java

```diff
@@ -35,14 +36,14 @@ public void testSingleUsePayloads() throws Exception {
         heartbeat.addSingleUsePayload("" + StatusPayloads.Type.IncomingDataDrop, "" + val2);
         final Status currentHeartbeatStatus = heartbeat.getCurrentHeartbeatStatus();
         List<Status.Payload> payloads = currentHeartbeatStatus.getPayloads();
-        Assert.assertEquals(2, payloads.size());
+        assertEquals(2, payloads.size());
         int value = 0;
         for (Status.Payload p : payloads) {
             if (StatusPayloads.Type.valueOf(p.getType()) == StatusPayloads.Type.IncomingDataDrop)
                 value = Integer.parseInt(p.getData());
         }
-        Assert.assertEquals(val2, value);
+        assertEquals(val2, value);
         payloads = heartbeat.getCurrentHeartbeatStatus().getPayloads();
-        Assert.assertEquals(1, payloads.size());
+        assertEquals(1, payloads.size());
```

Comment:
Lets log the payloads before asserting the size, so when the test fails, it's clearer why.

Reply:
Thanks for the suggestion. Fixed in 07ad13495b36c32f60850afc7b24e801fbe64a2e